packages/wonder-stuff-ci/src/mobile-release-git-utils.ts

```diff
@@ -0,0 +1,14 @@
+import execProm from "./exec-prom";
+
+/**
+ *
+ * @returns all release tags sorted creation time ascending
```

Comment:
**suggestion:** We use the imperative to describe a function, as well as type the returned values (and params, as necessary).
```suggestion
 * Get all mobile release tags.
 * 
 * @returns {Promise<Array<string>>} A promise of all release tags sorted by creation time ascending.
```

Also, this seems like it could very easily be turned into a more generic, `getAllMatchingTags` by moving the `ios|android|unified` filter to an argument. No change requested - just noting that if this starts needing to get reused in non-mobile contexts, we may see the need to change that.